src/tour/TourManager.ts

```diff
@@ -164,18 +164,28 @@ export default class TourManager {
   }
 
   private setHighlight(mask: IBoundingBox) {
+    const fullAppHeight = this.backdrop.ownerDocument.body.scrollHeight; // full page height including scroll bars
+    const fullAppWidth = this.backdrop.ownerDocument.body.scrollWidth; // full page width including scroll bars
+
+    // set the new height of the backdrop
+    this.backdrop.style.height = `${fullAppHeight}px`;
+    this.backdrop.style.width = `${fullAppWidth}px`;
+
+    // also consider the current scroll offset inside the window
+    const scrollOffset = self.scrollY;
+
     // @see http://bennettfeely.com/clippy/ -> select `Frame` example
     this.backdrop.style.clipPath = `polygon(
       0% 0%,
-      0% 100%,
-      ${mask.left}px 100%,
-      ${mask.left}px ${mask.top}px,
-      ${mask.left + mask.width}px ${mask.top}px,
-      ${mask.left + mask.width}px ${mask.top + mask.height}px,
-      ${mask.left}px ${mask.top + mask.height}px,
-      ${mask.left}px 100%,
-      100% 100%,
-      100% 0%
+      0% ${fullAppHeight}px,
+      ${mask.left}px ${fullAppHeight}px,
+      ${mask.left}px ${mask.top + scrollOffset}px,
```

Comment:
what happens when the user scrolls? or why is the scroll offset not considered when computing the mask?